<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlignmentAttrDialog</class>
 <widget class="QDialog" name="AlignmentAttrDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="alBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Alignment</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_textName">
        <property name="text">
         <string>Text name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edit_textName"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_source">
        <property name="text">
         <string>Source:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="edit_source"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_lastSync">
        <property name="text">
         <string>Last synced:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="lastSynced">
        <property name="text">
         <string>unknown</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_lastUpdate">
        <property name="text">
         <string>Last update:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="lastUpdate">
        <property name="text">
         <string>unknown</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ver1Box">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>First document</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_v1Name">
        <property name="text">
         <string>Version name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edit_v1Name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_v1Source">
        <property name="text">
         <string>Source:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edit_v1Source"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_v1LastSync">
        <property name="text">
         <string>Last synced:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="v1LastSynced">
        <property name="text">
         <string>unknown</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_v1LastUpdate">
        <property name="text">
         <string>Last update:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="v1LastUpdated">
        <property name="text">
         <string>unknown</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_v1Perm">
        <property name="text">
         <string>Permissions:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="permv1Editing">
          <property name="text">
           <string>editing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="permv1ChStruct">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>changing structure</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_v1IDStruct">
        <property name="text">
         <string>ID structure:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_1">
        <item>
         <widget class="QSpinBox" name="edit_v1Levels">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_v1Levels">
          <property name="text">
           <string>levels,</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_v1Prefix">
          <property name="text">
           <string>prefix: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_v1prefix"/>
        </item>
        <item>
         <widget class="QLabel" name="label_v1Separ">
          <property name="text">
           <string>separ.:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_v1Separ"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ver2Box">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Second document</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_v2Name">
        <property name="text">
         <string>Version name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edit_v2Name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_v2Source">
        <property name="text">
         <string>Source:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edit_v2Source"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_v2LastSync">
        <property name="text">
         <string>Last synced:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="v2LastSynced">
        <property name="text">
         <string>unknown</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_v2LastUpdate">
        <property name="text">
         <string>Last update:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="v2LastUpdated">
        <property name="text">
         <string>unknown</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_v2Perm">
        <property name="text">
         <string>Permissions:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="permv2Editing">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>editing</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="permv2ChStruct">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>changing structure</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_v2IDStruct">
        <property name="text">
         <string>ID structure:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="edit_v2Levels">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_v2Levels">
          <property name="text">
           <string>levels,</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_v2Prefix">
          <property name="text">
           <string>prefix: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_v2prefix"/>
        </item>
        <item>
         <widget class="QLabel" name="label_v2Separ">
          <property name="text">
           <string>separ.:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_v2Separ"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_warning">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AlignmentAttrDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AlignmentAttrDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
